crates/typst-library/src/math/matrix.rs

```diff
@@ -111,26 +106,23 @@ pub struct MatElem {
     #[default(HAlignment::Center)]
     pub align: HAlignment,
 
-    /// Draws augmentation lines in a matrix.
+    /// 行列内に補助線を描画。
     ///
-    /// - `{none}`: No lines are drawn.
-    /// - A single number: A vertical augmentation line is drawn
-    ///   after the specified column number. Negative numbers start from the end.
-    /// - A dictionary: With a dictionary, multiple augmentation lines can be
-    ///   drawn both horizontally and vertically. Additionally, the style of the
-    ///   lines can be set. The dictionary can contain the following keys:
-    ///   - `hline`: The offsets at which horizontal lines should be drawn.
-    ///     For example, an offset of `2` would result in a horizontal line
-    ///     being drawn after the second row of the matrix. Accepts either an
-    ///     integer for a single line, or an array of integers
-    ///     for multiple lines. Like for a single number, negative numbers start from the end.
-    ///   - `vline`: The offsets at which vertical lines should be drawn.
-    ///     For example, an offset of `2` would result in a vertical line being
-    ///     drawn after the second column of the matrix. Accepts either an
-    ///     integer for a single line, or an array of integers
-    ///     for multiple lines. Like for a single number, negative numbers start from the end.
-    ///   - `stroke`: How to [stroke]($stroke) the line. If set to `{auto}`,
-    ///     takes on a thickness of 0.05em and square line caps.
+    /// - `{none}`: 線は描画されません。
+    /// - 単一の数値: 指定された列番号の後に垂直方向の線を描画します。
+    /// 負数の場合は最後の列から数え始めます。
+    /// - 単一の辞書: 水平方向および垂直方向の両方で複数の補助線を描画できます。
```

Comment:
ここは`A single number`に対して`A dictionary`のため、単に「辞書」でよいかもしれません。

```suggestion
    /// - 辞書: 水平方向および垂直方向の両方で複数の補助線を描画できます。
```